src/solvers/flattening/boolbv_cond.cpp

```diff
@@ -12,6 +12,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 bvt boolbvt::convert_cond(const exprt &expr)
 {
+  PRECONDITION(expr.id() == ID_cond);
```

Comment:
Introduce a `cond_exprt` (whatever that is).

Reply:
`ID_cond` is never assigned to anything, I would say remove it

Reply:
It seems to be a list of guarded commands, but as @romainbrenguier mentioned it's not assigned anywhere. @kroening Is this still needed or can we remove `ID_cond` and `convert_cond`?